BridgeCareApp/BridgeCare/DataAccessLayer/InvestmentStrategies.cs

```diff
@@ -106,5 +98,36 @@ public void SetInvestmentStrategies(InvestmentStrategyModel data, BridgeCareCont
             }
             return;
         }
+        public void UpsertYearlyData(InvestmentStrategyModel investment,
```

Comment:
Upsert:  Replaces the yearly elements that exist with requested yearly elements. When the new data runs out it deletes the remainder of the array. If instead the requested yearly array is larger it fills what is there and then adds to the end. 
this is just a bug fix.

Reply:
ok